src/Http/Http.Abstractions/src/ProblemDetails/HttpValidationProblemDetails.cs

```diff
@@ -24,12 +24,9 @@ public HttpValidationProblemDetails()
     /// </summary>
     /// <param name="errors">The validation errors.</param>
     public HttpValidationProblemDetails(IDictionary<string, string[]> errors)
```

Comment:
This looks like it used to make a defensive copy of the dictionary and it no longer does.

Reply:
Not sure to understand the goal of the defensive copy here?

Reply:
It may just have been a way to change the type, but I thought it was worth thinking about.  This appears to be a public API, so anyone can pass in any dictionary they want.  If they subsequently add or remove errors, what behavior do we expect?  Formerly, we would have used a snapshot from construction time.

Reply:
Ok I understand. I can revert the old behavior if you want.

Reply:
Ok I understand. I'll revert the old behavior and keep the defensive copy.

Reply:
In the absence of other information, I'd put the copy back, but I'd accept @halter73's approval of not having it.

Reply:
Done